tsconfig.json

```diff
@@ -1,10 +1,11 @@
 {
   "compilerOptions": {
+    "jsx": "react",
     "baseUrl": ".",
     "paths": {
       "@assets/*": ["src/assets/*"],
       "@components/*": ["src/components/*"],
-      "@styles/*": ["src/styles/*"],
+      "@styles/*": ["src/styles/"],
```

Comment:
Why changing that line, also we usually locate styles in src folder, not assets, so please move it.